internal/provider/nomad/task_test.go

```diff
@@ -1,8 +1,9 @@
-package nomad
+package nomad_test
```

Comment:
Does not look related to your changes and I prefer to keep the same package name.

Reply:
Yea, this should be in a different patch. It must have snuck in after my nomad patch a while back.